src/i18n/Data/Intl/IntlLocales.php

```diff
@@ -1605,9 +1605,11 @@ public function validate($locale)
         if (!$lang || !$region) {
             return false;
         }
-        return strcasecmp($lang, $region)
```

Comment:
If you're happy with that! I don't know the context so thought this would suffice. Will re-push shortly.